server/config/config.js

```diff
@@ -1,31 +1,24 @@
 require('dotenv').config();
```

Comment:
not sure if its attempt  to take value from this scope or from parent scope, anyways we imported env in server, this should be removed